indy_node/server/config_req_handler.py

```diff
@@ -55,6 +57,20 @@ def _doStaticValidationPoolUpgrade(self, identifier, reqId, operation):
 
         # TODO: Check if cancel is submitted before start
 
+    def curr_pkt_info(self, pkg_name):
+        if pkg_name == "indy-node":
+            return Upgrader.getVersion(), ["indy-node"]
+        return NodeControlUtil.curr_pkt_info(pkg_name)
+
+    def get_dependencies(self, pkg_name, version):
+        base_deps = ["indy-node", "indy-plenum"]
+        if pkg_name == "indy-node":
+            return base_deps
+        deps = []
+        NodeControlUtil.dep_tree_traverse(
```

Comment:
Are we going to call 3d party tools to get dependencies during the static validation? I'm not sure that this is a good option...

Reply:
yes we are for now. if we do not check it here will need to check it during the upgrade and raise ex, but we just told that txn was validated and fine... I think it could be strange for user...
But I don't mind to remove it. Should I?